### PR TITLE
Button adjustments

### PR DESCRIPTION
**Pull request by sashako**
Includes:
+ removing underline on button link hover (as per design request)
+ create new button mixin. It makes it easier to generate stylesheets (see docs) and also will be quite useful as we add icon-support for buttons
+ add implementation info to the docs
+ update button shadows

### CODE REVIEW

File: docs/partials/buttons.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <div class="buttons-section">
   <h2 id="buttons" class="section-title">Buttons</h2>
-
+  <div class="use-info">
+    <h5>How to implement</h5>
+    <code>@include button($type, $size);</code>
```
Comment (nhinsch) [inline docs/partials/buttons.html]: Is this going to break the old mix-ins?

File: docs/styles/index.scss [hunk 2]
```diff
@@ -82,3 +83,45 @@ body {
   margin-bottom: 60px;
   overflow: hidden;
 }
+
+.use-info {
+  float: right;
+  margin-top: -100px;
+  background: $gray-lightest;
```
Comment (nhinsch) [inline docs/styles/index.scss]: Nit: Prefer `background-color`

File: docs/partials/buttons.html (hunk 1 above)
Comment (nhinsch) [inline docs/partials/buttons.html]: Never mind I see them below

**Comment by nhinsch** [comment]
Lgtm!

File: docs/partials/buttons.html (hunk 1 above)
Comment (sashako) [inline docs/partials/buttons.html]: nope, old mixins are still there. the only place we're using them is in omd login anyway. And i'm about to update those and then remove old mixins altogether

File: docs/styles/index.scss (hunk 2 above)
Comment (sashako) [inline docs/styles/index.scss]: good point

**Pull request merged**